### DISCUSSION TITLE
Strongswan licence for commercial use

### DISCUSSION THREAD

**Discussion by viplifes** [strongswan/strongswan#550] [General]
Hello!
We have a commercial company and we are seeking to integrate strongswan in our application.
We cannot publish our Android application to open source.
Is there any licence needed or agreement to be done in order to be legitimate for using your open source code?

**Comment by tobiasbrunner** [comment]
>  Is there any licence needed or agreement to be done in order to be legitimate for using your open source code?

Yes, a commercial license is required. Please contact us via email (I think you already have).

**Reply by ccccbjcn** [reply]
what is the email address?
